apps/src/gamelab/dropletConfig.js

```diff
@@ -23,12 +23,23 @@ var spriteBlockPrefix = 'sprite.';
 var groupBlockPrefix = 'group.';
 var animBlockPrefix = 'anim.';
 
-var GameLab;
+var gameLab;
 
 exports.injectGameLab = function (gamelab) {
-  GameLab = gamelab;
+  gameLab = gamelab;
 };
 
+// Generate a read-write property expansion function:
+function genPropExpansion (propname) {
+  return function (block) {
+    if (!block || block.type === 'socket') {
+      return propname;
+    } else {
+      return propname + ' = __;';
+    }
+  };
+}
+
```

Comment:
Is this something we can/should share? It looks identical between multiple files atm